internal/build/zone.go

```diff
@@ -184,16 +211,15 @@ func (z *Zone) IsTLD() bool {
 	return !strings.Contains(z.Domain, ".")
 }
 
-// Retire marks a zone as retired or withdrawn.
-func (z *Zone) Retire() {
```

Comment:
Removing this may mean we need to bump the major version of the library. Let's consider that separately from adding the new method, which we can keep.

Reply:
This is an internal package. Only this module can import it, hence we can break APIs anytime.

Reply:
Roger.